_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,110 @@
+---
+id: b8bb68
+name: Text spacing specified via CSS styles is below permissible minimum value or not set to `!important`
+rule_type: atomic
+description: |
+  This rule checks that
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing - Level AA
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][] and has any of the below CSS properties:
+
+- [word-spacing][]
+- [letter-spacing][]
+- [line-height][]
+
+## Expectation
+
+The target element does not have an overriding property value with `!important` specified for any of the above applicable CSS properties, except when value is below the permissible minimum respectively:
+
+**Note**: The permissible minimum values are `0.16em` for [word-spacing][], `0.12em` for [letter-spacing][], and `1.5em` for [line-height][] respectively.
+
+## Assumptions
+
+_There are currently no assumptions_
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 1.4.12: Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)
+- [CSS Text Module Level 3 - Spacing](https://www.w3.org/TR/css-text-3/#spacing)
+- [CSS Visual formatting model details](https://drafts.csswg.org/css2/visudet.html)
+
+## Test Cases
```

Comment:
I think this should have passing and failing example of each of the three properties. Have at least one where there are multiple props.